tests/ui/feature-gates/feature-gate-cfg-relocation-model.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(relocation_model = "pic")] //~ ERROR
+fn main() {}
```

Comment:
```suggestion
#[cfg(relocation_model = "pic")] //~ ERROR
fn foo() {}

fn main() {}
```
This should prevent the extra error about `main` missing.